src/modules/movieCounter.js

```diff
@@ -0,0 +1,8 @@
+const movieCounter = (count) => {
+  const counter = document.getElementById('counter');
+  counter.innerText = count;
+  return count;
+};
```

Comment:
- [ ] This function is not counting the number of items on the page. It is relying upon the data coming from the API and just updating the screen. According to the requirement, we should create a function that will count the number of items displayed on the page. 🤝️ One possible way to achieve this is by using the class name given for the individual movie `div`. I think you are using the class name `card` for an individual movie. 🧐️ You can use the output of the function to display the number.

**Example**
```
const movieCounter = () => {
  const movies = document.querySelectorAll('.card');
  return movies.length;
};
```

**Project requirements**
![Screenshot from 2023-04-19 19-44-45](https://user-images.githubusercontent.com/17243325/233656783-d5837312-7f8a-4da7-8b7c-46fc6c0c544b.png)